samples/package.json

```diff
@@ -12,6 +12,7 @@
     "gaxios": "^4.2.1"
   },
   "devDependencies": {
-    "mocha": "^8.0.0"
+    "mocha": "^8.0.0",
+    "node-polyfill-webpack-plugin": "^1.1.0"
```

Comment:
For the files below - please remove the `.DS_Store` files from the commit, and add `.DS_Store` to the `.gitignore` :)